### DISCUSSION TITLE
the finnale music

### DISCUSSION THREAD

**Discussion by werdkindom** [BEEmod/BEE2-items#4233] [Feature Suggestions]
i have one question why not have the finnale music base in beemod? i want it

**Comment by dsagfa** [comment]
Well for one, it's probably gonna be one singular phase's music (unless you get to choose what phase's music part to have).
For two, that's a bit of a big request, but I think they can do it.
For three, I would LOVE to have Bombs For Throwing At You in BEE2, if I could figure out how to make a boss fight in the PETI editor (if that is what it is called)
![aoldance](https://github.com/BEEmod/BEE2-items/assets/85970384/5eb0e258-e2aa-43a7-8147-ef0583ea785c)

**Reply by FrozenDragon0** [reply]
Puzzlemaker is not designed to make boss fight like stuff in it, use hammer if you want something like that.
